pkg/tagger/collectors/kubernetes_metadata_mapper.go

```diff
@@ -16,8 +16,11 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/kubernetes/apiserver"
 	"github.com/DataDog/datadog-agent/pkg/util/kubernetes/kubelet"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/hashicorp/golang-lru/simplelru"
 )
 
+const maxNamespacesInLRU = 100
```

Comment:
Pretty arbitrary limit right now, I didn't want to rely on the number of pods we fetch tags for as it can change.

Reply:
Got it thank you, could be useful to add a comment though.
Out of curiosity, any reason why you preferred using a LRU instead of the `cache` package we use in other parts of the agent? I'd expect the number of namespaces in a cluster to be almost constant.

Reply:
Sounds good, will add a comment. To clarify it a bit - the LRU is used within the call, and dropped once we return tags, the use of an LRU instead of the cache is due to the support for size limiting rather than time expiration. Also the choice of simple LRU as opposed to `github.com/hashicorp/golang-lru.LRU` - we don't need any syncronization, so intentionally using the LRU that is not thread unsafe. About the limit usage - we are sharing namespace labels across pods in the LRU and if we exceed the limit we'll start dropping the oldest entries, which can result in a new call to fetch labels for the same namespace during the same fetch of tags. Granted there are likely fewer namespaces in a cluster in most cases but if users are creating namespaces for some kind of testing, it can potentially become large.